deploy/roles/r-core/tasks/main.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Ensure umask is correct
+  lineinfile:
+    dest: "~/.bashrc"
+    line: umask 0022
+    regexp: (.*)umask(.*)
+    state: present
+
+- name: Install R-core
+  yum:
+    name: R-core
+    state: latest
+    disable_gpg_check: yes
+  register: r_core_updated
+  tags:
+    - r-core
+
+- name: Check if rstudio server is installed
+  yum:
+    list: rstudio-server
+  when: r_core_updated.changed
+  register: rstudio_installed
+  tags:
+    - r-core
+
+- name: Restart rstudio if R updates and rstudio server is installed
+  command: /bin/true
+  when: "{{ (rstudio_installed is defined and rstudio_installed.results|length > 0 and item['yumstate']=='installed')}}"
+  with_items: "{{ rstudio_installed.results | default([]) }}"
+  notify: restart rstudio-server
+  tags:
+    - r-core
+
+- name: Install base packages
+  yum:
+    name: "{{ item }}"
+    state: latest
```

Comment:
@lel99999 This should also use `state: "{{ r_core_state }}"` to make sure `R-devel` doesn't update `R-core` unintentionally.